crates/wdl-format/src/token/pre.rs

```diff
@@ -11,6 +11,44 @@ use crate::TokenStream;
 use crate::Trivia;
 use crate::TriviaBlankLineSpacingPolicy;
 
+/// Normalize single-line `#@ except:` directives
+fn normalize_except_directive(text: &str) -> String {
+    // Check if comment starts with #@
+    let text = text.trim();
+    if !text.starts_with("#@") {
+        return text.to_owned();
+    }
+
+    // Strip #@ prefix and get remainder
+    let remainder = text[2..].trim_start();
+
+    // Check if this is an except: directive
+    if !remainder.starts_with("except:") {
+        return text.to_owned();
+    }
+
+    // Check if single-line (no newlines in content)
+    if text.contains('\n') {
+        return text.to_owned();
+    }
+
+    // Extract the rule list after "except:"
+    let rules_text = &remainder[7..]; // Skip "except:"
```

Comment:
Same goes for here (note: needs to be `rustfmt`'d):

```suggestion
    let Some(rules_text) = remainder.trim_start().strip_prefix("except:") else { return text.to_owned(); }
    
    // Check if single-line (no newlines in content)
    if rules.contains('\n') {
        return text.to_owned();
    }
```